### DISCUSSION TITLE
How would you use @wagmi/cli with an existing NextJs project?

### DISCUSSION THREAD

**Discussion by zkSoju** [wevm/wagmi#1761] [General]
I'm running into errors with having `target: "es5"` in tsconfig.json. It works when removing it temporarily, but I get this error when running it.

` ERROR: Transforming const to the configured target environment ("es5") is not supported yet`

**Comment by tmm** [comment]
Bump it to `"es6"`

https://twitter.com/timneutkens/status/1608936340531613696?s=20
